runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/palette.json

```diff
@@ -5,6 +5,7 @@
     "Palette.GlobalConnectTool": "Globalne połączenie",
     "Palette.Create.StartEvent": "Utwórz Zdarzenie Początkowe",
     "Palette.Create.EndEvent": "Utwórz Zdarzenie Końcowe",
+    "Palette.Create.IntermediateEvent": "Create Zdarzenie Pośrednie",
```

Comment:
As far as I am concerned there is no word in polish like `create`

Reply:
xD